### DISCUSSION TITLE
is their any method or function to totally hide desktop wallet from web3Modal

### DISCUSSION THREAD

**Discussion by Sriharsh1103** [WalletConnect/walletconnect-monorepo#3034] [Developer Support] [answered]
i Don't want to display Metamask as recommended wallet but after implementing method like explorerRecommendedWalletIds: 'NONE' 
explorerExcludedWalletIds: 'ALL'

not any of them are working i want to totlly hide desktop wallet in web3Modal

**Comment by xzilja** [comment, marked as answer]
Options you specified above manage wallets fetched from our explorer api
https://docs.walletconnect.com/2.0/web/web3modal/react/wagmi/custom-wallets#custom-explorer-wallets

MetaMask (injected) comes from wagmi connectors, by default `w3mConnectors` includes WalletConnectConnector and InjectedConnector, but you can override this, just remove and add what you want like you normally do in Wagmi, however WalletConnectConnector is required
https://docs.walletconnect.com/2.0/web/web3modal/react/wagmi/custom-wallets#custom-wagmi-connectors
